From cae3cf5dbd894bb1a86d77f9d13af05d81edd6cd Mon Sep 17 00:00:00 2001
From: Pranav Vashi <neobuddy89@gmail.com>
Date: Wed, 14 Dec 2022 09:09:15 +0530
Subject: [PATCH 13/30] Launcher3: Improve spacing above and below dock
 searchbar

* Bottom of screen has some or other navbar components which adds little extra space

Change-Id: I82899a5546563259d3389459e7160c8eb6654a55
Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>
---
 res/values/config.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/res/values/config.xml b/res/values/config.xml
index cb6e50ba99..1b87ab7132 100644
--- a/res/values/config.xml
+++ b/res/values/config.xml
@@ -65,8 +65,8 @@
 
     <!-- Scalable Grid configuration -->
     <!-- This is a float because it is converted to dp later in DeviceProfile -->
-    <dimen name="hotseat_bar_bottom_space_default">48</dimen>
-    <dimen name="hotseat_qsb_space_default">0</dimen>
+    <dimen name="hotseat_bar_bottom_space_default">16</dimen>
+    <dimen name="hotseat_qsb_space_default">24</dimen>
 
     <!-- Whether to enable background preloading of task thumbnails. -->
     <bool name="config_enableTaskSnapshotPreloading">true</bool>
-- 
2.52.0

